go.mod

```diff
@@ -125,13 +125,17 @@ require (
 	golang.org/x/sys v0.0.0-20220412211240-33da011f77ad
 )
 
+require (
+	github.com/benbjohnson/clock v1.3.0
+	github.com/ipfs/go-log/v2 v2.5.1
+)
+
 require (
```

Comment:
WTF ?
I don't understand what `go.mod` is up to know

Reply:
I think there is a block for direct dependencies and a block for transitive/indirect dependencies